### DISCUSSION TITLE
My ReconstructMesh can only export .ply file but cannot export .mvs file

### DISCUSSION THREAD

**Discussion by c2k19zwh** [cdcseacave/openMVS#1115] [Q&A]
Hello everyone, I feel so sorry to bother you guys. But i am really in trouble, i need your help .

#OpenMVS x64 v2.3.0
#CPU: 12th Gen Intel(R) Core(TM) i5-12500H (16 cores)
#RAM: 15.31GB Physical Memory 2.00GB Virtual Memory
#OS: Linux 5.15.0-97-generic (x86_64)

when i running the command:
" ./ReconstructMesh -w /home/merry/OpenMVS/mvs_data/castle_data -i /home/merry/OpenMVS/mvs_data/castle_data/DensifyPointCloud.mvs -o ReconstructMesh.mvs  "
it can only export ReconstructMesh.ply, but cannot export ReconstructMesh.mvs.  I don't konw what the reason.
thanks all!
![2024-03-05 11-43-45 的屏幕截图](https://github.com/cdcseacave/openMVS/assets/147673646/acd90013-13ab-4629-a4b3-71621e686a0d)

**Comment by cdcseacave** [comment]
why do u need the mvs file?
see wiki on how to use openmvs

**Reply by ZJL-0422** [reply]
Hello, has the problem been resolved? I also encountered this issue when using version 2.3. How should I run the texturemesh command without generating scend_nse_mesh.mvs?

**Reply by cdcseacave** [reply]
See wiki pls

**Reply by xxxzzz200** [reply]
Sorry, I don't understand what you mean, from the wiki I see that the texturemesh command is to need scend_dense_mesh.mvs. but I can't generate scene_dense_mesh.ply right now, so what should I do.I think you realize that if a lot of people have the same problem, it may not be solved by just looking at the wiki.

**Comment by bowon95** [comment]
Hello 
I had the same matter and solved it.
 You need to put the option ' --archive-type 2 ' to the commend line and then you'll find .mvs file. Other stages too. If you need .mvs file, you have to put archive type option.

**Comment by cdcseacave** [comment]
no need for new MVS files for each module, now only one MVS file is used for all modules - read the readme and the wiki
